### PR TITLE
integrate book index and show with application template

### PR DESCRIPTION
**Pull request by fadyyousry**

### CODE REVIEW

File: app/helpers/book_helper.rb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 module BookHelper
-    def csv_names list
-        list.pluck(:name).join(',')
+    def csv_names(list)
+        list.pluck(:name).join(', ')
+    end
+
+    def human_boolean(boolean)
+        boolean ? 'Yes' : 'No'
     end
```
Comment (Magdyx) [inline app/helpers/book_helper.rb:8]: please don't use namings like : int, long, boolean, string
replace it with `flag`. 
or even better: make it as a checkbox or a switch button in the view

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -2,5 +2,7 @@ require("@rails/ujs").start()
 require("turbolinks").start()
 require("@rails/activestorage").start()
 require("channels")
-require('jquery')
-
+require('template/jquery.min.js')
+require('template/bootstrap.min.js')
+require('template/owl.carousel.min')
+require('template/custom')
```
Comment (Magdyx) [inline app/javascript/packs/application.js:8]: sorry what is `custom` file do?

File: app/helpers/book_helper.rb (hunk 1 above)
Comment (fadyyousry) [reply app/helpers/book_helper.rb:8]: localization :)

File: app/javascript/packs/application.js (hunk 2 above)
Comment (fadyyousry) [reply app/javascript/packs/application.js]: it's from template for carousel in book view I think

File: app/assets/stylesheets/template/magemenu.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
+@media all and (min-width: 992px) {
```
Comment (Magdyx) [inline app/assets/stylesheets/template/magemenu.css:1]: typo in file name

File: app/assets/stylesheets/template/magemenu.css [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+.megamenu {
+    border: none;
+    background-color: ivory;
+}
+
+.megamenu a:hover,
+.megamenu a:focus {
+  text-decoration: none;
+  color: white!important;
```
Comment (Magdyx) [inline app/assets/stylesheets/template/magemenu.css:16]: try not to use `!important` as much as possible

**Pull request merged**